README.md

```diff
@@ -250,7 +227,69 @@ Make sure to respect `YAML` identation properties, as the copied content
 must have some tabs/spaces to be correctly included under
 `gcloud-credentials.json`, the same way as the above example.
 
-## Uninstalling
+## Deploying to Your Cluster
+
+### Before Deploying
+
+Make sure you have [configured the operator](#configure-the-operator)
+correctly and have the following components installed in your machine before
```

Comment:
I would also mention here that a Docker registry account is required for pushing the container, giving Docker Hub as an example. I know that this should be self-explanatory for cloud-native people, but SD-WAN folks looking into CNWAN may need a bit more help with that side of things.

Reply:
Good catch! I am going to rebase and push force in a minute. Thank you!